### PR TITLE
Documents remapping esc (and other keys)

### PR DESCRIPTION
**Pull request by stsievert**
It does this based on issue #8. This also addresses #5, adding basic docs to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,27 @@ Jupyter.notebook.config.update({
 jupyter nbextension enable vim_binding/vim_binding
 ```
 
+## Custimization
+If you want to remap `<Esc>` to `jk`, we can insert the line ``CodeMirror.Vim.map("jk","<Esc>","insert")`` in `vim_binding.js` within the main function. For example,
```
Comment (lambdalisue) [inline README.md:61]: Rewriting `vim_binding.js` is strongly not recommended.

**Comment by lambdalisue** [comment]
Thanks for the PR. But I'm sorry that rewriting `vim_binding.js` is strongly not recommended so I don't really want to add that line. I haven't tested but use `custom.js` instead.

**Comment by stsievert** [comment]
You'll have to document that. A quick search returned some results but I don't have time to devote when I already have a working solution.

**Comment by lambdalisue** [comment]
I did. https://github.com/lambdalisue/jupyter-vim-binding#customize Thanks anyway for implementation :-)

**Pull request closed without merging**